build.gradle

```diff
@@ -121,12 +121,39 @@ bootJar {
 distributions {
     boot {
         contents {
+            from("$project.buildDir/libs/liquibase") {
+                include 'h2*.jar'
+                include 'liquibase-core*.jar'
+                include 'logback-*.jar'
+                include 'slf4j-api*.jar'
+                include 'snakeyaml*.jar'
+                into 'lib/liquibase'
+            }
+            from('src/main/resources/db/release-locks-changelog.xml') { into 'changelogs' }
             from('src/main/resources/email/images/') { into 'images/' }
             from('src/main/resources/email/templates/') { into 'templates/email' }
         }
     }
 }
 
+task copyToLib(type: Copy, dependsOn: [compileJava]) {
+    from findJar('h2')
+    from findJar('liquibase')
+    from findJar('logback-classic')
+    from findJar('logback-core')
+    from findJar('slf4j-api')
+    from findJar('snakeyaml')
+    into "${project.buildDir}/libs/liquibase"
+}
+
+def findJar(prefix) {
+    configurations.runtimeClasspath.filter {
+        it.name.startsWith(prefix)
+    }
+}
```

Comment:
This should take in an array of prefix's to prevent iterating through the same list multiple times.

Reply:
Will make the improvement in the master branch.